io/hdfs/src/test/java/org/apache/beam/io/hdfs/WritableCoderTest.java

```diff
@@ -14,7 +14,7 @@
  * the License.
  */
 
-package com.google.cloud.dataflow.contrib.hadoop;
+package org.apache.beam.io.hdfs;
```

Comment:
Noting that there is no such namespace as `java.jdk.util`. It is just `java.util`.